components/src/core/Drawer/DrawerNavItem.tsx

```diff
@@ -223,6 +223,10 @@ export const DrawerNavItem: React.FC<DrawerNavItemProps> = (props) => {
         // other View props
     } = otherProps;
 
+    const defaultProps: Partial<DrawerNavItemProps> = {
+        hidePadding: true,
```

Comment:
`hidePadding` does not have a default in this component. You are missing all of the props that do have default values.